### ISSUE TITLE
v3: thumbnails fluid & fixed are always null

### ISSUE THREAD

**Issue by mrgnou** [angeloashmore/gatsby-source-prismic#187]
Hi,

The following graphql query returns a null `prismicHome.data.thumbnails.portrait.fluid` node despite `prismicHome.data.thumbnails.portrait.url` is defined and correct. Same goes for fixed images.

```graphql
query MyQuery {
  prismicHome {
    data {
      hero {
        thumbnails {
          portrait {
            url
            fluid {
              ...GatsbyPrismicImageFluid
            }
          }
        }
      }
    }
  }
}
```

The schemas are up to date
gatsby-source-prismic version: 3.0.0-beta.19

**Comment by angeloashmore** [comment]
Hi @mrgnou, this should be resolved in the latest beta: `v3.0.0-beta.22`.

Could you let me know if this is still an issue after updating?

**Comment by mrgnou** [comment]
@angeloashmore Hey, I still have the same issue after updating to beta 24.
